src/runtime/virtcontainers/clh.go

```diff
@@ -944,7 +939,10 @@ func (clh *cloudHypervisor) hotplugAddBlockDevice(drive *config.BlockDrive) erro
 
 	// Hotplug block devices on PCI segments >= 1. PCI segment 0 is used
 	// for the network interface, any disks present at Guest boot time, etc.
-	clhDisk.SetPciSegment(int32(drive.Index)/32 + 1)
+	// Just bus 0 of each segment is used, and up to 31 devices can be
+	// plugged in to each bus.
```

Comment:
in to each segment?

Reply:
Yes:
- Segments (a.k.a. domains) contain buses. CLH supports a single bus (bus 0) per segment.
- A CLH bus contains 31 slots for virtual devices.